propagation/honeycomb.go

```diff
@@ -71,8 +76,9 @@ func UnmarshalHoneycombTraceContext(header string) (*PropagationContext, error)
 }
 
 // UnmarshalHoneycombTraceContextV1 takes the trace header, stripped of the
-// version string, and returns the component parts. Trace ID and Parent ID
-// are both required. If either is absent a nil trace header will be returned.
+// version string, and returns the component parts. If the header includes a
+// parent id but not a trace id, or if the header contains an unparseable
+// string in the trace context, an error will be returned.
 func UnmarshalHoneycombTraceContextV1(header string) (*PropagationContext, error) {
```

Comment:
I cannot, and I also think it should be unexported. Not in this release though, since that would be a breaking change. Lemme mark it as deprecated though!